estmator_project/est_quote/models.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.contrib.auth.models import User
+from est_client.models import Client
+
+
+@python_2_unicode_compatible
+class Category(models.Model):
+    name = models.CharField(max_length=256)
+
+    def __str__(self):
+        return 'Category: {}'.format(self.name)
+
+
+@python_2_unicode_compatible
+class Product(models.Model):
+    # Inputs
+    category = models.ForeignKey(Category)
+    name = models.CharField(max_length=256)
+
+    # Piece Multipliers
+    mins_piece = models.FloatField()
+    mult_dollies = models.FloatField()
+    m_cart = models.FloatField()
+    l_cart = models.FloatField()
+    p_cart = models.FloatField()
+    s_pack = models.FloatField()
```

Comment:
though names like this can be more convenient to write, understanding what they mean is dependent on inside-baseball.  What happens if the next dev in line to inherit this code base knows nothing about what the short names mean?  Is there documentation anywhere that helps de-mystify these names?  Why not give them more complete names instead so that documentation is not required?